block/params/block.go

```diff
@@ -0,0 +1,19 @@
+package params
```

Comment:
Not part of each block type?

Reply:
No: needed in a factory and in config.  (Of course, I _could_ still push each one out, but that would make the factory and the config have to import 3x as many params.)